.github/workflows/bazel.yml

```
@@ -67,6 +67,8 @@ jobs:
       - uses: actions/checkout@v4
         with: { fetch-depth: 0 }
 
+      - uses: bazelbuild/setup-bazelisk@v3
```

I think we should use https://github.com/bazel-contrib/setup-bazel instead

```suggestion
      - uses: bazel-contrib/setup-bazel@0.15.0
```

---

If you look at my commit history I tried that and didn't get it to work after 2 tries. The old way did work and I wanted to try to unblock quickly. I can keep poking if desired

---

Helps to read :facepalm: